spec/features/user_searches_for_a_beer_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+feature "As a user, I want to search for a beer", %{
+  [ ] I should be able to search for a beer
+  [ ] I must recieve an error message if beer doesn't exist
+  [ ] I should see an index of beers that match my search
+} do
+  scenario "successfully search for a brew" do
+    beer = FactoryGirl.create(:beer)
+    beer2 = FactoryGirl.create(:beer, name: "Piss")
```

Comment:
haha, uhh...  rename it to something other than this so if a hiring partner looks, they will assume you are not crazy.